cpu_usage.py

```diff
@@ -0,0 +1,35 @@
+#This script will check cpu usage in percentage
```

Comment:
Remove [this file](cpu_usage.py)

Reply:
script is not in root folder it is in advancedscripts

Reply:
Okay... Great✌🏼